eng/targets/CSharp.Common.props

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="MicroBuild.Core" Version="0.3.0" PrivateAssets="All" AllowExplicitReference="true" ExcludeAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' OR '$(RazorSdkCurrentVersionProps)' != ''">
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdkWeb)' == 'true' OR '$(UsingMicrosoftNETSdkBlazorWebAssembly)' == 'true' OR '$(RazorSdkCurrentVersionProps)' != ''">
```

Comment:
Why are we bringing back the `UsingMicrosoftNETSdkBlazorWebAssembly` property?

Reply:
This might be to prevent double imports?

Reply:
This is set in the SDK similar to other SDKs

* https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props#L24
* https://github.com/dotnet/sdk/blob/master/src/WebSdk/Web/Sdk/Sdk.props#L17